### PR TITLE
Remove import of associated publications from OncoKB

### PR DESCRIPTION
**Pull request by susannasiebert**

### CODE REVIEW

**Comment by susannasiebert** [comment]
This PR is ready for final review

File: lib/genome/online_updaters/oncokb/updater.rb [hunk 1]
```diff
@@ -50,7 +50,8 @@ def create_interaction_claims
                 drug_claim = create_drug_claim(drug['drugName'], drug['drugName'], 'OncoKB Drug Name')
                 interaction_claim = create_interaction_claim(gene_claim, drug_claim)
                 create_interaction_claim_attribute(interaction_claim, 'combination therapy', combination_drug_name)
-                add_interaction_claim_publications(interaction_claim, variant['pmids'])
+                #Our current agreement with OncoKB preclused us from importing the associated PMIDs
```
Comment (acoffman) [inline lib/genome/online_updaters/oncokb/updater.rb:53]: Minor typo here `s/preclused/precludes`

File: lib/genome/online_updaters/oncokb/updater.rb [hunk 2]
```diff
@@ -59,7 +60,8 @@ def create_interaction_claims
               drug = drugs[drug_name]
               drug_claim = create_drug_claim(drug['drugName'], drug['drugName'], 'OncoKB Drug Name')
               interaction_claim = create_interaction_claim(gene_claim, drug_claim)
-              add_interaction_claim_publications(interaction_claim, variant['pmids'])
+              #Our current agreement with OncoKB preclused us from importing the associated PMIDs
```
Comment (acoffman) [inline lib/genome/online_updaters/oncokb/updater.rb:63]: Minor typo here `s/preclused/precludes`

File: lib/genome/online_updaters/oncokb/updater.rb (hunk 1 above)
Comment (susannasiebert) [inline lib/genome/online_updaters/oncokb/updater.rb:53]:
```suggestion
                #Our current agreement with OncoKB precludes us from importing the associated PMIDs
```

File: lib/genome/online_updaters/oncokb/updater.rb (hunk 2 above)
Comment (susannasiebert) [inline lib/genome/online_updaters/oncokb/updater.rb:63]:
```suggestion
              #Our current agreement with OncoKB precludes us from importing the associated PMIDs
```

**Pull request merged**